dev-util/cmake/files/cmake-3.6.3-FindCUDA.patch

```diff
@@ -0,0 +1,27 @@
+--- a/Modules/FindCUDA/run_nvcc.cmake	2016-12-17 16:46:54.370587130 +0000
++++ b/Modules/FindCUDA/run_nvcc.cmake	2016-12-17 16:53:10.161588315 +0000
+@@ -104,14 +104,17 @@
+ # Add the build specific configuration flags
+ list(APPEND CUDA_NVCC_FLAGS ${CUDA_NVCC_FLAGS_${build_configuration}})
+ 
++# The following may cause an nvcc fatal: redefenition of compiler-bindir
++if(NOT @CMAKE_GENTOO_BUILD@ STREQUAL ON AND NOT @CMAKE_BUILD_TYPE@ STREQUAL Gentoo )
```

Comment:
Well, from what you're saying it seems that it's a correct upstream bug, not accounting for the `=` variant. If it's valid for nvcc, you might consider at least opening a bug to CMake people to draw their attention to it. However, if I were you, I wouldn't care enough to do that ;-D.